includes/rest-api/class-wp-rest-stories-controller.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+/**
+ * Class AMP_REST_Stories_Controller
+ */
+class AMP_REST_Stories_Controller extends WP_REST_Posts_Controller {
+	/**
+	 * Prepares a single story for create or update. Add post_content_filtered field to save/insert.
+	 *
+	 * @param WP_REST_Request $request Request object.
+	 *
+	 * @return stdClass|WP_Error Post object or WP_Error.
+	 */
+	protected function prepare_item_for_database( $request ) {
+		$prepared_story = parent::prepare_item_for_database( $request );
+
+		if ( is_wp_error( $prepared_story ) ) {
+			return $prepared_story;
+		}
+
+		if ( isset( $request['content_filtered'] ) ) {
+			$prepared_story->post_content_filtered = $request['content_filtered'];
+			if ( isset( $request['content_filtered']['raw'] ) ) {
+				$prepared_story->post_content_filtered = $request['content_filtered']['raw'];
+			}
+			$prepared_story->post_content_filtered = maybe_serialize( $prepared_story->post_content_filtered );
+		}
+
+
+		return $prepared_story;
+	}
+
+	/**
+	 * Prepares a single story output for response. Add post_content_filtered field to output.
+	 *
+	 *
+	 * @param WP_Post $post Post object.
+	 * @param WP_REST_Request $request Request object.
+	 *
+	 * @return WP_REST_Response Response object.
+	 */
+	public function prepare_item_for_response( $post, $request ) {
+		$response = parent::prepare_item_for_response( $post, $request );
+		$fields   = $this->get_fields_for_response( $request );
+		$data     = $response->get_data();
+		$schema   = $this->get_item_schema();
+
+		if ( in_array( 'content_filtered', $fields, true ) ) {
+			$post_content_filtered    = maybe_unserialize( $post->post_content_filtered );
+			$data['content_filtered'] = array(
+				'raw' => rest_sanitize_value_from_schema( $post_content_filtered, $schema['properties']['content_filtered']['properties']['raw'] ),
+			);
+		}
+
+		$context = ! empty( $request['context'] ) ? $request['context'] : 'view';
+		$data    = $this->filter_response_by_context( $data, $context );
+		$links   = $response->get_links();
+		// Wrap the data in a response object.
+		$response = rest_ensure_response( $data );
+		foreach ( $links as $rel => $rel_links ) {
+			foreach ( $rel_links as $link ) {
+				$response->add_link( $rel, $link['href'], $link['attributes'] );
+			}
+		}
+
+		/**
+		 * Filters the post data for a response.
+		 *
+		 * The dynamic portion of the hook name, `$this->post_type`, refers to the post type slug.
+		 *
+		 *
```

Comment:
Remove line.

Reply:
Fixed in https://github.com/ampproject/amp-wp/pull/3871/commits/00846a7433a5909b8efe6d671f302965efa4484f